healvis/simulator.py

```diff
@@ -385,15 +506,18 @@ def run_simulation(param_file, Nprocs=1, sjob_id=None, add_to_history=''):
         param_dict = param_file
```

Comment:
we should do a deepcopy here to prevent accidental inplace operations further down.

Reply:
I don't think we actually overwrite any entries in the `param_dict`, but it doesn't hurt to be extra careful.